src/fakeData/fakeHealthcareProfessionals.ts

```diff
@@ -23,12 +23,20 @@ export function generateRandomCreateHealthcareProfessionalInputArray({ count = 5
 }
 
 function generateLocalizedNameInput(): gqlTypes.LocalizedNameInput {
-    return {
-        firstName: faker.person.firstName(),
-        middleName: faker.person.middleName(),
-        lastName: faker.person.lastName(),
-        locale: faker.helpers.enumValue(gqlTypes.Locale)
-    }
+    const randomLocal = faker.helpers.enumValue(gqlTypes.Locale)
```

Comment:
In the previous tests every field of names was random. This caused validations errors since it created undesirable names. For example:
```
  {
    firstName: 'John',
    middleName: 'Emerson',
    lastName: 'Doe',
    locale: 'ja_JP'
  }
```